libs/test-setup/src/lib.rs

```diff
@@ -444,3 +463,38 @@ pub async fn create_postgres_database(original_url: &Url) -> Result<Quaint, AnyE
 
     Ok(conn)
 }
+
+pub async fn create_mssql_database(jdbc_string: &str) -> Result<Quaint, AnyError> {
+    let mut splitted = jdbc_string.split(';');
+    let uri = splitted.next().unwrap().to_string();
+
+    let mut params: BTreeMap<String, String> = splitted
```

Comment:
This thing will be replaced when @yoshuawuyts gets the connection-string crate ready. We need proper parsing, and this will break really bad when having escaped semicolons `(;)` somewhere in the values.